installation-linux/README.md

```diff
@@ -53,14 +54,14 @@ Running the following commands will install meeko and ringtail in
 the currently active micromamba environment.
 
 ```
-pip install meeko==0.6.0a3
 pip install ringtail
+pip install meeko==0.6.0a3
 ```
 
 Running the following commands should display help messages:
 ```
 mk_prepare_ligand.py
-rt_process_vs.py
+rt_process_vs -h
```

Comment:
Without the -h it will return an error (help message still visible but we may want to just remove the error message)

Reply:
Good point, just opened https://github.com/forlilab/Ringtail/issues/57